src/devices/heatit.ts

```diff
@@ -0,0 +1,14 @@
+import {Definition} from '../lib/types';
+import extend from '../lib/extend';
+
+const definitions: Definition[] = [
+    {
+        fingerprint: [{type: 'Router', manufacturerName: 'Heatit Controls AB', modelID: 'Dimmer-Switch-ZB3.0'}],
+        model: 'Zig Dim 250W',
```

Comment:
Is there any better model number written on the device/product site?

Reply:
Hi, this is actually the official name

https://www.thermo-floor.no/heatit-controls/1444420/heatit-zig-dim-250w-zigbee-rotary-dimmer-white-ral-9003

pdf manual

https://documents.heatit.com/1444420/datasheet/eng?_ga=2.217320981.262970775.1690485878-1179559807.1690485878

Open for other suggestions